docs/tutorials/control/air-quality-fleet.md

```diff
@@ -0,0 +1,953 @@
+---
+title: "Monitor Air Quality with a Fleet of Sensors"
+linkTitle: "Air Quality Fleet"
+type: "docs"
+description: "Use a fleet of machines with air quality sensors to monitor PM 2.5 levels in different indoor and outdoor locations."
+images: ["/tutorials/air-quality-fleet/three-sensor-dash-wide.png"]
+imageAlt: "A web dashboard showing PM2.5 readings from two air quality sensors."
+tags: ["tutorial"]
+authors: ["Jessamy Taylor"]
+languages: ["typescript"]
+viamresources: ["sensor", "data_manager"]
+level: "Intermediate"
+date: "2024-04-23"
+# updated: ""  # When the tutorial was last entirely checked
+cost: 200
+# Learning goals:
+# 1. The reader can distinguish the concepts of organizations and locations and can select the appropriate setup when creating their own projects for their business.
+# 2. The reader can identify when to use fragments and evaluate when it is worth using fragments.
+#    The reader can create their own fragments for their projects and knows what to include and exclude from them.
+# 3. The reader recognizes how permissions enable the management of data for a business across multiple customers while providing each customer access to their own data.
+---
+
+In this tutorial you will use a fleet of devices to collect air quality data from different places and display the most recent readings from each device in a custom viewing dashboard.
+By completing this project, you will learn to:
+
+- Configure a fleet of identical machines
```

Comment:
I was hesitant to say this since they don't know what fragments are yet, and might increase cognitive load. But I can see this being helpful to folks who are specifically looking for a tutorial that covers this fragments use case so I guess okay to include?